### PR TITLE
chore: remove bootnodes from chainspecs

### PR DESCRIPTION
**Pull request by DylanVerstraete**
# Description of proposed changes

Removed hardcoded local bootnodes from chainspecs

---

Practical tips for PR review & merge:

- [x] All GitHub Actions report PASS
- [x] Newly added code/functions have unit tests
  - [ ] Coverage tools report all newly added lines as covered
  - [ ] The positive scenario is exercised
  - [ ] Negative scenarios are exercised, e.g. assert on all possible errors
  - [ ] Assert on events triggered if applicable
  - [ ] Assert on changes made to storage if applicable
- [x] Modified behavior/functions - try to make sure above test items are covered
- [ ] Integration tests are added if applicable/needed

### CODE REVIEW

File: chainspecs/devnetSpecRaw.json [hunk 1]
```diff
@@ -2,9 +2,7 @@
   "name": "Creditcoin3 Dev",
   "id": "creditcoin3_dev",
   "chainType": "Live",
-  "bootNodes": [
-    "/ip4/127.0.0.1/tcp/30333/p2p/12D3KooWFEX4ncCfVv2VqCZLTaPoGTuXQR4pY3M2aGV1eGSqpmUQ"
-  ],
+  "bootNodes": [],
```
Comment (atodorov) [inline chainspecs/devnetSpecRaw.json:5]: Do you mind adding a check which makes sure that there are no boot nodes defined in any of the chain-specs ?

Option 1 - add it somewhere here: 
https://github.com/gluwa/creditcoin3/blob/dev/.github/workflows/ci.yml#L91

can be a simple grep/bash script or maybe slightly better parsing the json with `jq`.

Option 2  - we have a dedicated section for chainspec checks which executes one of the existing integration tests with a different config file: 
https://github.com/gluwa/creditcoin3/blob/dev/.github/workflows/ci.yml#L687


Either one will work but 1) should fail faster so is probably better.

**Review by atodorov — CHANGES_REQUESTED** [review]
Can you squash both commits together ? No need to have bootnodes added and then removed in the history IMO.

File: chainspecs/devnetSpecRaw.json (hunk 1 above)
Comment (DylanVerstraete) [reply chainspecs/devnetSpecRaw.json:5]: Implemented 1

**Review by beqaabu — DISMISSED** [review]
Nice

**Review by atodorov — CHANGES_REQUESTED** [review]
LGTM but the last commit isn't expected. I think this PR needs a rebase and the commit [Auto-update TypeScript type definitions](https://github.com/gluwa/creditcoin3/pull/276/commits/2b36411b249c3fddddba9b345da9ad09a4d70bb3) should be removed.

**Review by atodorov — APPROVED** [review]
LGTM, waiting for CI jobs to report green before merging.

**Review by beqaabu — APPROVED** [review]

**Pull request merged**